common/views/components/Promo/Promo.js

```diff
@@ -24,67 +24,45 @@ function partOf(commissionedSeries, seriesTitle) {
 }
 
 function contentTypeLabel(contentType, commissionedSeries) {
-  // FIXME: this hack can die when we separate out a WorkPromo
-  return contentType !== 'work' && (
+  return (
     <span className={`line-height-1 bg-yellow absolute promo__content-type ${font({s: 'HNM5'})} ${spacing({s: 1}, {padding: ['top', 'bottom', 'left', 'right']})}`}
       aria-hidden='true'>{commissionedSeries ? 'Digital Story' : contentType.charAt(0).toUpperCase() + contentType.slice(1)}</span>
   );
 }
 
-function titleFontClasses(contentType) {
-  if (contentType === 'work') {
-    return `${font({s: 'HNL5'})} text--truncate`;
-  } else {
-    return font({s: 'WB5'});
-  }
-}
-
 function headingText(title, contentType) {
   if (title) {
     return title;
-  } else if (contentType === 'work') {
-    return 'Title not found';
   } else {
     return 'Coming soon…';
   }
 }
 
 type PromoDescriptionProps = {|
-  standalone: boolean,
   children: React.Node
 |}
 
-const PromoDescription = ({standalone, children}: PromoDescriptionProps) => {
-  if (standalone) {
-    return (
-      <div className={`row bg-cream ${spacing({s: 10}, {padding: ['top']})}`}>
-        <div className='container'>
-          <div className='grid'>
-            <div className={`${grid({l: 10, shiftL: 1, m: 10, shiftM: 1, s: 12})}`}>
-              {children}
-            </div>
-          </div>
-        </div>
-      </div>
-    );
-  } else {
-    return (
-      <Fragment>{children}</Fragment>
-    );
-  }
+const PromoDescription = ({children}: PromoDescriptionProps) => {
```

Comment:
What's the point in this?

Reply:
Hangover from what was here before, but now the Promo has less logic in it, this abstraction isn't needed. I'll remove it.

Reply:
Yea, let's keep a eye out for as-and-when refactoring to avoid having a tech dept sprint.